includes/payments/functions.php

```diff
@@ -161,7 +161,8 @@ function edd_insert_payment( $payment_data = array() ) {
 			'currency'     => $payment_data['currency'],
 			'downloads'    => $payment_data['downloads'],
 			'user_info'    => $payment_data['user_info'],
-			'cart_details' => $payment_data['cart_details']
+			'cart_details' => $payment_data['cart_details'],
+			'tax'          => $cart_tax,
```

Comment:
This was intentionally removed so that we can cut down on the amount of meta data stored. It's stored in its own key now.

Looks like we need to fix our backwards compat rather than re-store this.